.github/workflows/eden_gcp.yml

```diff
@@ -0,0 +1,78 @@
+---
+name: EdenGCP
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: [master]
+# yamllint disable rule:line-length
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: get eden
+        uses: actions/checkout@v2
+      - name: setup packages
+        run: |
+          sudo apt update
+          sudo apt install -y qemu-utils openvpn
+          echo "$OVPN_FILE" | base64 -d > ./config.ovpn
+        env:
+          OVPN_FILE: ${{ secrets.OVPN_FILE }}
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: Public IP
```

Comment:
Can you please move this step to be right before we divulge our first secret? (setup packages)

Reply:
Fixed